tests/queries/0_stateless/01881_join_on_conditions_hash.sql.j2

```diff
@@ -30,7 +30,6 @@ SELECT t1.key, t1.key2 FROM t1 INNER ALL JOIN t2 ON t1.id == t2.id AND t2.key ==
 
 SELECT '--';
 SELECT t1.key FROM t1 INNER ANY JOIN t2 ON t1.id == t2.id AND t2.key == t2.key2 AND t1.key == t1.key2;
-SELECT t1.key FROM t1 INNER ANY JOIN t2 ON t1.id == t2.id AND t2.key == t2.key2 AND t1.key == t1.key2 AND 0; -- { serverError INVALID_JOIN_ON_EXPRESSION,NOT_FOUND_COLUMN_IN_BLOCK }
```

Comment:
why the queries are just removed in that test?
Do they pass now or still fail?

Reply:
I think most of deleted queries covered by another tests, but it's hard to sort out each one and find specific case in another test, so I restored this deleted cases.